package.json

```diff
@@ -26,8 +26,10 @@
     "yargs": "^13.3.0"
   },
   "devDependencies": {
+    "@types/graphql": "^14.5.0",
     "@types/jest": "^24.0.15",
-    "@types/lodash": "^4.14.136",
+    "@types/lodash": "^4.14.162",
+    "@types/lodash.find": "^4.6.6",
```

Comment:
Its kinda hard to find the correct lodash typings to import, this was the only way I could make it work 😓